src/markdown-pages/instant-observability/build-a-quickstart/create-a-dashboard.mdx

```diff
@@ -0,0 +1,138 @@
+---
+path: '/instant-observability/build-a-quickstart/create-a-dashboard'
+title: 'Create a dashboard'
+template: 'GuideTemplate'
+description: 'create a dashboard to observe your data in New Relic'
+duration: '15 min'
+---
+
+<Callout variant="course" title="Lab">
+
+This procedure is a part of lab that teaches you how to build a quickstart. If you haven't already, checkout the [_Lab introduction_](/instant-observability/build-a-quickstart)
+
+Each procedure in this lab builds on top of the last one, so make sure you [_deploy your application_](/instant-observability/build-a-quickstart/deploy-your-application) before proceeding with this one.
+
+</Callout>
+
+New Relic One dashboards allow you to quickly observe and make meaning of the information that you collect. You can gather and chart the specific data you want to see and customize it the way you want to see.
+
+You're going to create dashboard in New Relic One. Navigate to https://one.newrelic.com/ and sign in with your account. Here, you can see different tabs such as **Browse Data**, **Dashboards**, **Alerts & AI** etc. 
+Move to **Dashboards** and click **Create a dashboard** in the top right corner. 
+
+![create a dashboard](../../../images/instant-observability/create-dashboard.png)
+
+Name your dashboard **FlashDB** and click create.
+
+![name your dashboard](../../../images/instant-observability/name-dashboard.png)
+
+## Add charts to your dashboard
+
+Once the dashboard is in place, you can start creating charts. You're going to add the folloiwng charts:
+- Database methods
+- Avergae response time
+- Errors
+- Database size
+- Cache hits
+- Keys
+
+Hover over the dashboard and click **Add a new chart**.
+
+![add a new chart](../../../images/instant-observability/add-new-chart.png)
+
+New Relic One allow you to add charts using query builder or you can choose to add text, images or links using Markdown. Click **Add a chart**.
+
+![add a chart](../../../images/instant-observability/add-a-chart.png)
+
+It will take you to a query builder option. Edit the query as follow:
+
+>```SQL
+>SELECT count(*) FROM fdb_method FACET method 
+>```
+
+![database methods chart](../../../images/instant-observability/fdb-methods.png)
+
+Here, you group your results of FlashDB methods by attribute values and see the count of each type of transaction. You can choose to present your results in different formats such as table, billboard, pie chart etc. 
```

Comment:
```suggestion
Here, you see the count of each FlashDB query, grouped by method. You can choose to present your results in different formats such as table, billboard, or pie chart. 
```

A few suggestions here:

1. The query reads left to right as SELECT, FROM, FACET, and the text reads nicer if it follows the same pattern.
2. "such as" implies that you're not giving the entire list, so you don't need etc.
3. Add the Oxford comma, per style guide